src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/mbscheduler_hwemu.cxx

```diff
@@ -2222,6 +2222,67 @@ namespace hwemu {
         return 0;
     }
 
+    int xocl_scheduler::convert_execbuf(xocl_cmd* xcmd)
+    {
+        size_t src_off;
+        size_t dst_off;
+        size_t sz;
+        uint64_t src_addr = -1; 
+        uint64_t dst_addr = -1; 
+        struct ert_start_copybo_cmd *scmd = xcmd->ert_cp;
+
+        /* CU style commands must specify CU type */
+        if (scmd->opcode == ERT_START_CU || scmd->opcode == ERT_EXEC_WRITE)
+            scmd->type = ERT_CU;
+
+        /* Only convert COPYBO cmd for now. */
+        if (scmd->opcode != ERT_START_COPYBO)
+            return 0;
+
+        sz = ert_copybo_size(scmd);
+
+        src_off = ert_copybo_src_offset(scmd);
+        xclemulation::drm_xocl_bo* sBo = device->xclGetBoByHandle(scmd->src_bo_hdl);
+
+        dst_off = ert_copybo_dst_offset(scmd);
+        xclemulation::drm_xocl_bo* dBo = device->xclGetBoByHandle(scmd->dst_bo_hdl);
+
+        if(!sBo && !dBo)
+        {
+            return -EINVAL;
+        }

```

Comment:
Better no braces for single statement, just follow the single statement with a new line.